### DISCUSSION TITLE
How to prevent redirect after collection update (Edit collection)

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#11384] [Archive]
Hi @rijkvanzanten 
How to prevent redirect after update the collection? I want stay on same edit collection after save button clicked.

Thanks
Sandeep

**Comment by abdonrd** [comment]
You have the `Save and stay` option:

<img width="296" alt="Screenshot 2022-02-02 at 09 48 50" src="https://user-images.githubusercontent.com/1007051/152121710-01e40f56-3f15-4f21-96ce-e646fb860098.png">

**Reply by sandyksit** [reply]
@abdonrd ... dots is not displaying , Is there any configuration?

**Reply by abdonrd** [reply]
Edit anything to make the dots appear.

**Reply by sandyksit** [reply]
dots is not appearing for me.

**Reply by abdonrd** [reply]
Are you using the latest version of Directus?

**Reply by benhaynes** [reply]
@sandyksit — post a screenshot so we know you're in the right place.

**Reply by sandyksit** [reply]
I am using 9.2.1

**Reply by licitdev** [reply]
@sandyksit The `Save and Stay` functionality is not implemented in `/settings/data-model`. It is however implemented for the `/content` module when you create / update items within a collection.
